lemur/notifications/cli.py

```diff
@@ -85,12 +85,10 @@ def security_expiration_summary(exclude):
     status = FAILURE_METRIC_STATUS
     try:
         print("Starting to notify security team about expiring certificates!")
-        success, failed = send_security_expiration_summary(exclude)
+        status = send_security_expiration_summary(exclude)
         print(
-            "Finished notifying security team about expiring certificates! "
-            f"Sent: {success} Failed: {failed}"
+            "Finished notifying security team about expiring certificates!"
         )
-        status = SUCCESS_METRIC_STATUS
```

Comment:
don't we still need this, when `status==True`?
https://github.com/Netflix/lemur/blob/56169656377637b7a0e53514105fbe31b436c4a9/lemur/constants.py#L12

also should this method return false: 
https://github.com/Netflix/lemur/blob/f941383cd2cee0bf8c11350cc39f45cabd44502f/lemur/notifications/messaging.py#L420

Reply:
Oops, you're right, I was getting `SUCCESS_METRIC_STATUS` and `True` confused, will fix.

Re: "should this method return false" - the existing pattern for the file seems to be return only `True`, and nothing on failure. It's a bit of an odd pattern, but I don't want to rewrite the whole thing now and I'd like to stay consistent with the rest of the methods in the file.